### PR TITLE
feat: add command line chic

### LINKED ISSUE #36477: Update the color scheme to freeCodeCamp chic
This is a master issue to track all the color scheme changes to various parts of the platform main.

### PR DESCRIPTION
**Pull request by ahmaxed**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #36477

### CODE REVIEW

File: client/src/components/Map/components/Block.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ export class Block extends Component {
     });
     return (
       <li className={`block ${isExpanded ? 'open' : ''}`}>
-        <button
+        <div
```
Comment (xyozio) [inline client/src/components/Map/components/Block.js]: Why changed to `div`?

Comment (ahmadabdolsaheb) [reply client/src/components/Map/components/Block.js]: I will revert that. thank you for noticing it.

**Comment by ahmadabdolsaheb** [comment]
Thank you all for reviewing this pr. Here are some considerations.

The following views and components will be either removed or redesigned therefore they are not optimized ref #36621 #36480, #36479, #36478:
- About
- Support
- Sponsors
- Academic Honesty
- Code of Conduct
- Privacy Policy
- Terms of Service
- landing page
- welcome page
- top navigation
- search bar
- search page

@ValeraS, thank you for your recommendation about naming convention. I will make sure to adopt one before merging this pr.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
I am merging this, and we should make further changes in separate PRs.

**Pull request merged**